### PR TITLE
19836 change buttons by specifications

### PR DESCRIPTION
**Pull request by vcostin**
https://community.openproject.org/work_packages/19836

### CODE REVIEW

File: app/views/version_settings/edit.html.erb [hunk 1]
```diff
@@ -38,5 +38,5 @@ See doc/COPYRIGHT.rdoc for more details.
 
 <% labelled_tabular_form_for :version, @version, :url => project_version_path(@project, @version), :html => {:method => :put} do |f| %>
 <%= render :partial => 'form', :locals => { :f => f } %>
-<%= submit_tag l(:button_save), class: 'button -highlight' %>
+<%= styled_button_tag l(:button_save), class: '-success -with-icon icon-yes' %>
```
Comment (vcostin) [inline app/views/version_settings/edit.html.erb]: Where is this button?

**Comment by floriank** [comment]
@vcostin @myabc with the new findings in the discussion around the buttons, there should be no need for this anymore, I think.

**Comment by vcostin** [comment]
@floriank no more changes in buttons in plugins?

**Comment by myabc** [comment]
@floriank @vcostin no, that's not accurate: We _do_ want the icons. We _don't_ want the colour change. I'm sorry for communicating this really badly :(

@vcostin please take a look at #2991. We will only use "green" buttons in the work packages toolbar, at least for now.

**Pull request merged**